shell/lisa_shell

```diff
@@ -31,7 +31,9 @@ LISA_PYTHON=${LISA_PYTHON:-python3}
 export LISA_DEVMODE=${LISA_DEVMODE:-1}
 
 # Setup colors
-source "$LISA_HOME/shell/lisa_colors"
+if [ -z $LISA_SHELL_QUIET ]; then
```

Comment:
Can we use the same "pattern" we already have for `LISA_DEVMODE` ?

```python
export LISA_SHELL_QUIET=${LISA_SHELL_QUIET:-0}
#...
if [[ $LISA_SHELL_QUIET == 1]]; then
else
fi
```

Reply:
Yes, much better. That should be done now.